src/components/Home/Home.js

```diff
@@ -0,0 +1,791 @@
+  bookAndContributeContainerSVG: {
+    color: '#400CCC',
+  },
+
+  carouselContainerComponents: {
+    width: '320px',
+    display: 'flex',
+    flexShrink: 1,
+    height: '300px',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    padding: '0px 20px',
+    '@media (max-width: 799px)': {
+      width: '300px',
+      height: '300px',
+    },
+  },
+
+  carouselComponentParagraph: {
+    textAlign: 'left',
+    fontFamily: 'Open Sans, sans-serif',
+    fontWeight: 400,
+    fontSize: '16px',
+    color: '#000000',
+    '@media (max-width: 799px)': {
+      marginBottom: '0px',
+    },
+  },
+  inputForm: {
+    height: '45px',
+    border: 'none',
+    width: '435px',
+    borderRadius: '4px',
+    padding: '0px 15px',
+    marginBottom: '10px',
+    marginRight: '10px',
+    '@media (max-width: 799px)': {
+      marginBottom: '10px',
+      width: '90%',
+      marginRight: '0px',
+    },
+    '&::placeholder': {
+      fontFamily: ' Open Sans',
+      fontSize: '16px',
+    },
+  },
+  slackFormContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+    justifyContent: 'flex-start',
+    '@media (max-width: 799px)': {
+      width: '80%',
+      alignItems: 'center',
+    },
+  },
+  slackForm: {
+    width: '100%',
+    '@media (max-width: 799px)': {
+      alignItems: 'center',
+    },
+  },
+}));
+
+const theme = createMuiTheme({
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 799,
+      md: 960,
+      lg: 1280,
+      xl: 1920,
+    },
+  },
+});
+
+export default function Home() {
+  const matches = useMediaQuery(theme.breakpoints.down('sm'));
+
+  const {
+    root,
+    introContainer,
+    topImageContainer,
+    topImage,
+    bookAndContributeContainer,
+    workathonContainer,
+    ourMissionContainer,
+    carouselContainer,
+    joinSlackContainer,
+    title,
+    getConnectedHeader,
+    bookAndContributeComponentTitle,
+    paragraph,
+    firstContainerDiv,
+    workathonSVG,
+    meetOnSlackSVG,
+    registerWebmaking,
+    letGrow,
+    workathonContainerDiv,
+    bookAndContributeComponents,
+    bookAndContributeContainerParagraph,
+    ourMissionComponent,
+    ourMissionComponentParagraph,
+    titleWhite,
+    mentorSVG,
+    volunteerSVG,
+    partnerSVG,
+    greenCircle,
+    bookMentorsContainer,
+    bookAndContributeContainerSVG,
+    carouselContainerComponents,
+    carouselComponentParagraph,
+    inputForm,
+    slackFormContainer,
+    slackForm,
+  } = useStyles();
+
+  return (
+    <ThemeProvider theme={theme}>
+      <div className={root}>
+        <Grid container>
+          <Grid className={introContainer} item xs={12} sm={6}>
+            <div className={firstContainerDiv}>
+              <Typography variant="h1" className={getConnectedHeader}>
+                Hey Girl, Get Connected!
+              </Typography>
+              <Typography
+                variant="body1"
+                className={paragraph}
+                paragraph={true}
```

Comment:
use es6 if attribs is more 2, for example:

<Typography {...{variant: 'body', className: paragraph, paragraph: true }}>